source/contributing/implementation/mixins.rst

```diff
@@ -21,49 +21,116 @@ the `SpongeCommon repository <https://github.com/SpongePowered/SpongeCommon/tree
 are extensively documented and cover many of the more complex scenarios. You should also consult the Javadoc of the Mixin
 repository itself, since almost everything is already documented.
 
-**Caveat: When contributing mixins, note that you cannot use anonymous classes.**
+.. note::
+
+  The Mixin project will be servicing a number of other projects in addition to Sponge itself. Therefore Mixin has its'
+  own documentation together with the repository.
+
+
+Mixins and Inner Classes
+========================
+
+While you can use lambdas, along with anonymous and inner classes, inside mixins, you cannot use an anonymous/inner class within another
```

Comment:
Too many commas. May I suggest the following edit:
"While you can use lambdas, anonymous and inner classes inside mixins, you cannot use an anonymous/inner class within another ..."